submissions/emmanuelEsuong/htmlForm/signupPage.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>SignUp</title>
+</head>
+
+<body>
+    <nav><pre>
+
+        <a href="file:///C:/Users/TOSHIBA/Documents/GitHub/Dufuna-CodeCamp20/submissions/emmanuelEsuong/htmlForm/signupPage.html">SignUp</a>
+
+        <a href="file:///C:/Users/TOSHIBA/Documents/GitHub/Dufuna-CodeCamp20/submissions/emmanuelEsuong/htmlTask/index.html">Product List</a>
+
+        <a href="file:///C:/Users/TOSHIBA/Documents/GitHub/Dufuna-CodeCamp20/submissions/emmanuelEsuong/htmlForm/htmlForm.html">Contact Us</a>
+
+    </nav></pre>
+
+    <br>
+
+    <h1>SignUp:</h1>
+
+    <form action="#" method="POST" enctype="multipart/form-data">
```

Comment:
There should be just form tag here. Remove the other elements i.e `action`, `method` and `enctype`